packages/common/core/src/form/method.controller.ts

```diff
@@ -64,6 +63,10 @@ class OdsCommonFieldMethodController<T extends OdsCommonFieldAttribute & OdsComm
     }
   }
 
+  async setTabindex(value: number): Promise<void> {
```

Comment:
has been removed for all other, do we need to add it again for this one to work?